config/locales/exemptions/en.yml

```
@@ -130,14 +130,14 @@ en:
 
     MISC_EXEMPTIONS: &MISC_EXEMPTIONS
       no_respondent_address_html: |
-        <span class="gv-u-heading-small">You don’t have sufficient contact details for the other people in this application (the respondents)</span><br/>
+        <span class="gv-u-heading-small">You don’t have sufficient contact details for the other people in this application (the respondents)</span>
         <span class="form-hint">The other people can’t be contacted to arrange a MIAM.</span>
       without_notice_html: |
         <span class="gv-u-heading-small">You’re applying for a without notice hearing</span><br/>
```

I think we can get rid of this `<br/>` too and that would be all of them 😃 
```suggestion
         <span class="gv-u-heading-small">You’re applying for a without notice hearing</span>
```